CHANGELOG-1.2.1.md

```diff
@@ -0,0 +1,5 @@
+# Changelog since v1.2.0
```

Comment:
Instead of adding a new file, can we prepend it to the top of the CHANGELOG-1.2.md file?

Reply:
Sure